tests/httpServer/cors.test.ts

```diff
@@ -250,15 +250,15 @@ describe("http-cors-tests", () => {
   });
 });
 
-class TestEndpointsDefCORS {
+export class TestEndpointsDefCORS {
```

Comment:
Seems that the old way still works (backward compatible) -- if I just want to test a subset of classes I can provide them into the list.

Maybe still keep a test to ensure this compatibility?

Reply:
Sure, maybe I revert one with a big whitespace change :-D

Reply:
Reverted the 2 with the whitespace changes, so now the PR is smaller and those test the backward compatibility.